raiden/network/proxies/utils.py

```diff
@@ -33,3 +41,36 @@ def compare_contract_versions(
             f'Provided {contract_name} contract ({to_normalized_address(address)}) '
             f'version mismatch. Expected: {expected_version} Got: {deployed_version}',
         )
+
+
+def get_onchain_locksroots(
+        raiden,
+        token_network_address: TokenNetworkAddress,
+        channel_identifier: ChannelID,
+        participant1: Address,
+        participant2: Address,
+        block_identifier: BlockSpecification,
+) -> Tuple[Locksroot, Locksroot]:
+    payment_channel = raiden.chain.payment_channel(
+        canonical_identifier=CanonicalIdentifier(
+            chain_identifier=None,
```

Comment:
Why is chain ID None here? What does it mean? I am not sure how this canonical identifier is supposed to be used yet. But I see there are some constants here: https://github.com/raiden-network/raiden/blob/adb2cbd984dfb86f719e4365f2b135c8d8545381/raiden/utils/__init__.py#L29

Reply:
I believe that you can do this when you pass a canonical identifier where the called function doesn't intend use this & its somewhat impossible / hard / too much work to get the chain ID and set it in the first place.

Reply:
I will import this constant and use it instead of None